pkgs/applications/audio/airwindows/default.nix

```
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "airwindows-${version}";
+  version = "2020-05-25";
```

If you package something based on a commit instead of a release, you need to add `unstable` to the name.

Whether to put it in `pname` or `version` has been discussed in https://github.com/NixOS/nixpkgs/pull/88023 with no consensus, but I think putting it here should for now be better than omitting it completely.
```suggestion
  version = "unstable-2020-05-25";
```

---

Done.